app/assets/javascripts/discourse/app/components/topic-footer-buttons.js

```diff
@@ -13,22 +15,30 @@ export default Component.extend({
   // Allow us to extend it
   layoutName: "components/topic-footer-buttons",
 
-  @discourseComputed("topic.isPrivateMessage")
+  @discourseComputed("topic.isinlineActionablesPrivateMessage")
```

Comment:
Looks like you didn't add this in this PR, but it's not properly camelCased. `isInline` - the `I` should be capital.